src/zenml/client.py

```diff
@@ -1205,6 +1209,234 @@ def register_pipeline(
         )
         raise AlreadyExistsException(error_msg)
 
+    def export_pipeline_runs(self, filename: str) -> None:
+        """Export all pipeline runs to a YAML file.
+
+        Args:
+            filename: The filename to export the pipeline runs to.
+
+        Raises:
+            RuntimeError: If no pipeline runs exist.
+        """
+        import json
+
+        from zenml.utils.yaml_utils import write_yaml
+
+        pipeline_runs = self.zen_store.list_runs(
+            project_name_or_id=self.active_project.id
+        )
+        if not pipeline_runs:
+            raise RuntimeError("No pipeline runs found.")
+        yaml_data = []
+        for pipeline_run in pipeline_runs:
+            pipeline_run.status = self.zen_store.get_run(pipeline_run.id).status
```

Comment:
Why do we need an additional call to `get_run` here? Isn't `pipeline_run.status` the same thing that we got back from `list_runs`?

Reply:
Initially no, but now yes :D

Good catch, I'll remove the call.